### DISCUSSION TITLE
Defining canvas borders for the Drawing add-on

### DISCUSSION THREAD

**Discussion by akantifirst** [mozman/ezdxf#594] [Q&A] [answered]
I've noticed that when exporting dxf to pdf with Matplotlib backend, the borders of ~5% of layout size are added. 
Is it possible to avoid these borders?
![2021-12-26 14_59_57-output - PDF-XChange Editor](https://user-images.githubusercontent.com/82590770/147410409-67ac5e29-147a-4409-ad37-e8cccc031ceb.png)

**Comment by mozman** [comment]
Take a look at the FAQ for the `drawing` add-on #550

**Comment by akantifirst** [comment, marked as answer]
I was able to remove margins by using ax.margins method:

 
 ​        ​ax​: ​plt​.​Axes​ ​=​ ​fig​.​add_axes​((​0​, ​0​, ​1​, ​1​)) 
 ​        ​ax​.​margins​(​0​, ​0​)
